include/nbl/builtin/hlsl/spirv_intrinsics/raytracing.hlsl

```diff
@@ -13,33 +13,47 @@ namespace hlsl
 namespace spirv
 {
 
+using RayQueryKHR = vk::SpirvOpaqueType<spv::OpTypeRayQueryKHR>;
+using AccelerationStructureKHR = vk::SpirvOpaqueType<spv::OpTypeAccelerationStructureKHR>;
+
+// matching Ray Query Committed Intersection Type
+static const uint32_t RayQueryCommittedIntersectionNoneKHR = 0;
+static const uint32_t RayQueryCommittedIntersectionTriangleKHR = 1;
+static const uint32_t RayQueryCommittedIntersectionGeneratedKHR = 2;
+
+// matching Ray Query Candidate Intersection Type
+static const uint32_t RayQueryCandidateIntersectionTriangleKHR = 0;
+static const uint32_t RayQueryCandidateIntersectionAABBKHR = 1;
```

Comment:
aren't there enums in `spv::` namespace and SPIR-VHeadeers for this ?

Reply:
I tried
```
[[vk::ext_builtin_input(spv::RayQueryCommittedIntersectionNoneKHR)]]
static const uint32_t RayQueryCommittedIntersectionNoneKHR;
```
on godbolt. Couldn't compile as it couldn't find it

Reply:
Also couldn't find it in `spirv.hpp`

Reply:
> I tried
> 
> ```
> [[vk::ext_builtin_input(spv::RayQueryCommittedIntersectionNoneKHR)]]
> static const uint32_t RayQueryCommittedIntersectionNoneKHR;
> ```
> 
> on godbolt. Couldn't compile as it couldn't find it

because its not a builtin, builtins are things like `gl_LocalInvocationIndex`

If I had to guess, `RayQueryCommittedIntersectionNoneKHR` is just an enum/literal or something

Reply:
there's literally an enum for this: https://github.com/KhronosGroup/SPIRV-Headers/blob/main/include/spirv/unified1/spirv.hpp#L1260